### PR TITLE
そんな感じのPR

### PR DESCRIPTION
**Pull request by aikimatsu**

### CODE REVIEW

File: test-branch [hunk 1]
```diff
@@ -1,3 +1,3 @@
-test
-test
-test
+text
+text
+text
```
Comment (aikimatsu) [inline test-branch]: leave a comment

File: test-branch [hunk 2]
```diff
@@ -1,3 +1,3 @@
-test
-test
-test
+text
+text
```
Comment (aikimatsu) [inline test-branch]: aaaaa

Comment (aikimatsu) [inline test-branch]: single comment

File: test-branch (hunk 1 above)
Comment (tsuchiya-fido) [reply test-branch]: Great

File: test-branch (hunk 2 above)
Comment (tsuchiya-fido) [reply test-branch]: My meeting today from 8am has just been postponed to next week.  Lucky!

Comment (tsuchiya-fido) [reply test-branch]: How are you doign?! WE had snow yesterday and it is too cold till noon today. But after that, the temprature will be up to 18 degree.  Crazy weather today...

Comment (aikimatsu) [reply test-branch]: ちとコメントの加え方の確認をしてみていました。
See you tomorrow in ARTF.

**Pull request merged**